### DISCUSSION TITLE
No support response at all: GitHub Actions is currently disabled for your account

### DISCUSSION THREAD

**Discussion by XInTheDark** [community/community#48936] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

This message banner appears on top of the "Actions" tab of a repository:

<img width="941" alt="image" src="https://user-images.githubusercontent.com/61100393/222292342-4198e108-2d60-4cc9-be35-a20f890dc5ec.png">

It prompted me to reach out to GitHub support, which I immediately did. **But I have not received a response for 4 weeks!** Is there any way the staff can process this faster? It is hindering development for open source projects. 

For reference, my ticket number is #1996586.

**Comment by mspist** [comment]
I have the same issue. I opened the ticket 2110192, but I hadn't any answer.
